docs/js/react-dom.js

```diff
@@ -37,7 +37,7 @@
  * This source code is licensed under the BSD-style license found in the
  * LICENSE file in the root directory of this source tree. An additional grant
```

Comment:
In the generated version there are `spaces`. And my IDE deleted them because of the rule `trim_trailing_whitespace` in` .editorconfig`